packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -616,30 +627,31 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   ]);
 
   /* @conditional-compile-remove(rich-text-editor-composite-support) */
-  const richTextSendBoxOptions = useMemo(() => {
+  const richTextSendBoxOptions: RichTextSendBoxOptions | undefined = useMemo(() => {
     return options?.richTextEditor
       ? {
           /* @conditional-compile-remove(rich-text-editor-image-upload) */
           ...richTextEditorOptions,
           /* @conditional-compile-remove(rich-text-editor-image-upload) */
-          onInsertInlineImage: (imageUrl: string, imageFileName: string) => {
+          onInsertInlineImage: (imageAttributes: Record<string, string>, imageFileName?: string) => {
             onInsertInlineImageForSendBox(
-              imageUrl,
-              imageFileName,
+              imageAttributes,
+              imageFileName || 'image.png',
```

Comment:
Should we store this default filename in a constants file?

We use twice in this file

We also use it in CopyPastePlugin.tsx